instrumentation/apache-httpasyncclient-4.1/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpasyncclient/ApacheHttpAsyncClientInstrumentation.java

```diff
@@ -149,6 +149,10 @@ public void resetRequest() throws IOException {
     public void close() throws IOException {
       delegate.close();
     }
+
+    public Context getContext() {
```

Comment:
can you add a comment here about why this method exists? I'm worried someone (e.g. me) will remove it since it's unused

Reply:
+1 added a comment. 

There will be probably more instrumentations that do not expose the context which makes it impossible for extending instrumentations to access the current span.